cluster-provision/k8s/1.18/provision.sh

```diff
@@ -2,19 +2,22 @@
 
 set -ex
 
+[ -f "/tmp/pre-pull-images" ] || exit 1
```

Comment:
i would suggest to support please an env var that skip the pre pulling
its good when one debug provisioning locally, and no need to pull images
it saves couple minutes of provisioning, when you need to do faster iterations

Reply:
For local debugging you just can remove the file containing the images (1.18/pre-pull-images i.e.)

Reply:
but here wont it exit 1 if the file wont be copied to /tmp?
would be glad please if it can be tested end to end